src/example/solution.js

```diff
@@ -65,7 +65,7 @@ const initialState = {
 
 function Timer() {
   const [state, dispatch] = useReducer(reducer, initialState, init);
-  let timer;
+  const timer = useRef(null);
```

Comment:
If you want to _keep alive_ a value inside a hooks after each cycle, you have to use `useRef`,

Per [React docs](https://reactjs.org/docs/hooks-reference.html#useref):

> Essentially, useRef is like a “box” that can hold a mutable value in its .current property.

Reply:
Great, thanks @alejandronanez